Fix resizing too small causing an index out of range crash

coolguy1842 opened the pull request:
I haven't verified the change too much, but it seems to work well.

Commit:
simple fix for resizing too small causing an index out of range crash

Signed-off-by: coolguy1842 <coolguy1842.deport391@passfwd.com>

Trial97 commented on launcher/ui/instanceview/VisualGroup.cpp:
```diff
@@ -66,6 +66,9 @@ void VisualGroup::update()
             rows[currentRow].height = maxRowHeight;
             rows[currentRow].top = offsetFromTop;
             currentRow++;
+            if(currentRow >= rows.size()) {
```
```suggestion
            if (currentRow >= rows.size()) {
```

coolguy1842 replied:
Hey, I thought I did but ended up doing it wrongly, Will try to fix now.

Trial97 commented:
And you need to sign your commits  see https://github.com/PrismLauncher/PrismLauncher/pull/2474/checks?check_run_id=25876625758

coolguy1842 commented:
Hey, I thought I did but ended up doing it wrongly, Will try to fix now.

Commit:
fix formatting

Signed-off-by: coolguy1842 <kian.goslett@protonmail.com>

Commit:
DCO Remediation Commit for coolguy1842 <kian.goslett@protonmail.com>

I, coolguy1842 <kian.goslett@protonmail.com>, hereby add my Signed-off-by to this commit: 448ded2387025b11d122d51b0abd907bec0452aa

Signed-off-by: coolguy1842 <kian.goslett@protonmail.com>
